tests/gpu/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+find_package(GLEW REQUIRED)
+find_package(GLUT REQUIRED)
+
```

Comment:
The code should add the OpenGL dependency.

Reply:
Good catch! I'll add that

Reply:
Fixed all instances of this